### ISSUE TITLE
PHP notice on client project

### ISSUE THREAD

**Issue by tareiking** [humanmade/batcache#22]
We haven't yet investigated the cause here, but noting the error as it's seen in x-ray logs currently. May lead to project specific implementation. Will cycle back with findings when we dig in further.

```
E_NOTICE: Trying to access array offset on value of type bool
/usr/src/app/vendor/humanmade/batcache/advanced-cache.php:577
```

**Comment by tcrsavage** [comment]
Same issue here

Can't replicate the issue locally after enabling local caching
The same code is hit, but the $batcache->cache['max_age'] value is correctly set in this case, and no warning is produced

**Comment by roborourke** [comment]
Fixed as of v1.3.5
